### ISSUE TITLE
Asking for a feature

### ISSUE THREAD

**Issue by Sark-07** [Devui-io/Devui_BugsAndReports#19]
Can the feature like when you click on the bars icon at the top right corner of the navbar in mobile mode, the menu container appears from the left side and also the menus will appear one after one but in a smooth speed be added to the navbar components?

**Comment by anirudhuuu** [comment]
@Sark-07 can you share an example for this?

**Comment by Sark-07** [comment]
> 

Yeah sure here is the link: https://react-mini-projects-beryl.vercel.app/
And here is the github link: https://github.com/Sark-07/React-mini-projects/tree/master/Navbar
This is build by me.

**Comment by Sark-07** [comment]
> @Sark-07 can you share an example for this?

@jwala-anirudh
Yeah sure here is the link: https://react-mini-projects-beryl.vercel.app/
And here is the github link: https://github.com/Sark-07/React-mini-projects/tree/master/Navbar
This is build by me.
